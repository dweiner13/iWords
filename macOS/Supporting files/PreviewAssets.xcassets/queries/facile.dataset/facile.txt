facil.e              ADJ    3 2 NOM S N POS             
facil.e              ADJ    3 2 VOC S N POS             
facil.e              ADJ    3 2 ABL S X POS             
facil.e              ADJ    3 2 ACC S N POS             
facilis, facile, facilior -or -us, facillimus -a -um  ADJ  
easy, easy to do, without difficulty, ready, quick, good natured, courteous;
facile               ADV    POS                         
facile, facilius, facillime  ADV  
easily, readily, without difficulty; generally, often; willingly; heedlessly;

