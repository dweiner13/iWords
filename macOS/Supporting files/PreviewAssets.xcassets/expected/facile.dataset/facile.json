[{"words":[{"meaning":"easy, easy to do, without difficulty, ready, quick, good natured, courteous;","expansion":{"adj":{"_0":"facilis, facile, facilior -or -us, facillimus -a -um","_1":[]}}}],"truncated":false,"possibilities":[{"adjective":{"_0":{"number":"S","gender":"N","variety":2,"degree":"POS","text":"facil.e","declension":3,"case":"NOM"}}},{"adjective":{"_0":{"number":"S","gender":"N","variety":2,"degree":"POS","text":"facil.e","declension":3,"case":"VOC"}}},{"adjective":{"_0":{"declension":3,"case":"ABL","number":"S","variety":2,"degree":"POS","text":"facil.e"}}},{"adjective":{"_0":{"number":"S","gender":"N","variety":2,"degree":"POS","text":"facil.e","declension":3,"case":"ACC"}}}]},{"words":[{"meaning":"easily, readily, without difficulty; generally, often; willingly; heedlessly;","expansion":{"adv":{"_0":"facile, facilius, facillime","_1":[]}}}],"truncated":false,"possibilities":[{"adverb":{"_0":{"degree":"POS","text":"facile"}}}]}]